tests/Functional/MenuTree/MenuTreeTraverserTest.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Tests\Functional\MenuTree;
+
+use DAMA\MenuBundle\Menu\MenuFactoryInterface;
+use DAMA\MenuBundle\Node\Node;
+use Symfony\Bundle\FrameworkBundle\Client;
+use Symfony\Bundle\FrameworkBundle\Test\WebTestCase;
+
+class MenuTreeTraverserTest extends WebTestCase
```

Comment:
actually I don't think we need this separate test :blush: 

You can just do the same assertions also on the DOM crawler inside `FunctionalTest`?

Reply:
Ok